packages/perseus-editor/src/editor-page.tsx

```diff
@@ -287,7 +296,7 @@ class EditorPage extends React.Component<Props, State> {
                     <ItemEditor
                         ref={this.itemEditor}
                         itemId={this.props.itemId}
-                        question={this.props.question}
+                        question={convertDeprecatedWidgets(this.props.question)}
                         answerArea={this.props.answerArea}
```

Comment:
This could be a spicy move -- but it seems to work with editing, and I can't move the conversion logic any higher.